### PR TITLE
Feat/bottom bar

### PR DESCRIPTION
**Pull request by Megavex**

### CODE REVIEW

File: application/app/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#Thu Sep 26 21:37:18 CEST 2019
```
Comment (cajoho99) [inline application/app/gradle/wrapper/gradle-wrapper.properties]: This file should not be included in the pull request

File: application/app/gradlew [hunk 2]
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
Comment (cajoho99) [inline application/app/gradlew]: This should not be included

File: application/app/gradlew.bat [hunk 3]
```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (cajoho99) [inline application/app/gradlew.bat]: Do not include

File: application/app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -9,8 +9,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+
```
Comment (cajoho99) [inline application/app/src/main/AndroidManifest.xml]: Remove new line

File: application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java [hunk 5]
```diff
@@ -10,13 +22,56 @@
 
 public class HomeActivity extends AppCompatActivity {
 
+    ImageButton btnCreate;
```
Comment (cajoho99) [inline application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java]: Add access modifier?

File: application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java [hunk 6]
```diff
@@ -10,13 +22,56 @@
...
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_home);
+
+        BottomNavigationView bottomnav = findViewById(R.id.bottom_navigation);
```
Comment (cajoho99) [inline application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java]: camelCase

File: application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java [hunk 7]
```diff
@@ -10,13 +22,56 @@
...
+        bottomnav.setOnNavigationItemSelectedListener(navListener);
+
+        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container,new HomeFragment()).commit();
+
+        btnCreate = findViewById(R.id.btn_create);
+
+        btnCreate.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Toast.makeText(HomeActivity.this, "test", Toast.LENGTH_SHORT).show();
```
Comment (cajoho99) [inline application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java]: Should this be included?

File: application/app/src/main/res/drawable/btn_background.xml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+
```
Comment (cajoho99) [inline application/app/src/main/res/drawable/btn_background.xml]: remove newlines

File: application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java (hunk 6 above)
Comment (Megavex) [reply application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java]: Where?

Comment (Megavex) [reply application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java]: bottom_navigation är en android file name standard, dunno if we wanna mess with it

Comment (cajoho99) [reply application/app/src/main/java/com/yahyeet/boardbook/activity/HomeActivity.java]: bottomnav borde vara bottomNav
